mlrun/datastore/datastore_profile.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import ast
+import base64
+import json
+import typing
+from urllib.parse import ParseResult, urlparse, urlunparse
+
+import pydantic
+
+import mlrun
+import mlrun.errors
+
+from ..secrets import get_secret_or_env
+
+
+class privateValue(pydantic.BaseModel):
```

Comment:
```suggestion
class PrivateValue(pydantic.BaseModel):
```
(since this is python, not Go :)